project.mml

```diff
@@ -531,10 +531,14 @@ Layer:
             way,
             landuse,
             military,
+            amenity,
+            "natural",
             way_area/NULLIF(POW(!scale_denominator!*0.001*0.28,2),0) AS way_pixels
           FROM planet_osm_polygon
           WHERE (landuse = 'military'
             OR military = 'danger_area')
+            OR ("natural" = 'scrub'
```

Comment:
This line is also not needed - you just want to render prisons as an overlay, not all natural=scrub.